### PR TITLE
Fix GCS history retrieval

### PR DESCRIPTION
**Pull request by ewiseblatt**
@duftler

### CODE REVIEW

File: front50-gcs/src/main/java/com/netflix/spinnaker/front50/model/GcsStorageService.java [hunk 1]
```diff
@@ -44,6 +44,7 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import java.util.ArrayList;
+import java.util.Comparator;
```
Comment (duftler) [inline front50-gcs/src/main/java/com/netflix/spinnaker/front50/model/GcsStorageService.java]: Sort order is off here.

File: front50-gcs/src/main/java/com/netflix/spinnaker/front50/model/GcsStorageService.java [hunk 2]
```diff
@@ -351,13 +364,13 @@ private void writeLastModified(String daoTypeName) {
       try {
         obj_api.update(bucketName, object.getName(), object).execute();
       } catch (HttpResponseException e) {
-        log.error("writeLastModified failed to update: {}", e.toString());
-
+          log.error("writeLastModified failed to update {}\n{}", timestamp_path, e.toString());
```
Comment (duftler) [inline front50-gcs/src/main/java/com/netflix/spinnaker/front50/model/GcsStorageService.java]: Indentation

File: front50-gcs/src/main/java/com/netflix/spinnaker/front50/model/GcsStorageService.java [hunk 3]
```diff
@@ -315,6 +319,15 @@ void storeObject(String objectKey, String daoTypeName, T obj) {
       log.error("Could not fetch versions from Google Cloud Storage: {}", e);
       return new ArrayList<>();
     }
+
+    Comparator<T> comp = (T a, T b) -> {
+        // reverse chronological
```
Comment (duftler) [inline front50-gcs/src/main/java/com/netflix/spinnaker/front50/model/GcsStorageService.java]: Reading on a phone, but aren't we using 2-space indentation?

**Comment by duftler** [comment]
Lgtm after addressing comments.

File: front50-gcs/src/main/java/com/netflix/spinnaker/front50/model/GcsStorageService.java (hunk 3 above)
Comment (ewiseblatt) [inline front50-gcs/src/main/java/com/netflix/spinnaker/front50/model/GcsStorageService.java]: ok. I was using an emacs that is configured differently and didnt notice.

**Pull request merged**